static/bidder-info/thetradedesk.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://direct.adsrvr.org/bid/bidder/{{.SupplyId}}"
+maintainer:
+  email: "Prebid-Maintainers@thetradedesk.com"
+gvlVendorID: 21
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  # User sync is supported but it requires configuration. Reach out to a TAM to get a <CookiePartnerId>
+  # Here is an example of how the url should look like:
+  # https://match.adsrvr.org/track/rid?ttd_pid=<CookiePartnerId>&fmt=json&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
```

Comment:
Do you support redirect and/or iframe? You should specify the supported format(s) as such:
```
userSync:
  # TheTradeDesk supports user syncing, but requires configuration by the host. Contact this
  # bidder directly at the email address in this file to ask about enabling user sync.
  supports:
    - redirect
    - iframe
```

I'm also not sure what a TAM is. I suggest adding a comment similar to what I provided in the snippet above directing hosts on how to reach out to you to configure syncing.

I think you can also omit the example URL as hosts can't do anything with it and it might cause confusion.

Reply:
Changed to match snippet.